xbmc/music/MusicDatabase.cpp

```diff
@@ -7458,14 +7452,14 @@ bool CMusicDatabase::GetAlbumsByWhereJSON(
                   StringUtils::Split(record->at(1 + i).get_asString(), ",");
               if (values.size() % 2 == 0) // Must contain an even number of entries
               {
-                for (size_t i = 0; i + 1 < values.size(); i += 2)
+                for (size_t j = 0; j + 1 < values.size(); j += 2)
                 {
-                  int idGenre = atoi(values[i].c_str());
+                  int idGenre = atoi(values[j].c_str());
```

Comment:
not really applicable but this could be `std::stoi` then you can drop the `c_str()`

Reply:
Want the silent fail of `atoi` not the exception raised by `stoi`